package.json

```
@@ -180,6 +180,15 @@
                     "type": "boolean",
                     "description": "%AWS.configuration.enableCodeLenses%",
                     "default": true
+                },
+                "aws.togglePrompts": {
+                    "type": "object",
+                    "description": "%AWS.setting.togglePrompts.description%",
```

other localization ids in this same file, use the `AWS.configuration.*` namespace. What is the reason for choosing `AWS.setting` (not used anywhere in the codebase)?
```suggestion
                    "description": "%AWS.configuration.togglePrompts.description%",
```

---

corrected